File: backend/services/post-ts-koa-mongo-fdm/src/common/api/invalid-text-post-content.ts
import { ErrorBody } from "../utils/error";

export const empty: ErrorBody = {
    error: {
        error: "TEXT_POST_CONTENT_EMPTY",
        reason: "Text post content cannot be empty",
        message: "Text post content cannot be empty",
    }
};
export const tooLong: ErrorBody = {
    error: {
        error: "TEXT_POST_CONTENT_TOO_LONG",
        reason: "text post content is too long",
        message: "text post content is too long",
    }
};

File: backend/services/post-python-fastapi-mongo-fdm/src/api/invalid_text_post_content.py
from typing import Final
from utils.error import ErrorBody

empty: Final[ErrorBody] = {
    "error": {
        "error": "TEXT_POST_CONTENT_EMPTY",
        "reason": "Text post content cannot be empty",
        "message": "Text post content cannot be empty",
    }
}

tooLong: Final[ErrorBody] = {
    "error": {
        "error": "TEXT_POST_CONTENT_TOO_LONG",
        "reason": "Text post content is too long",
        "message": "Text post content is too long",
    }
}
